cmd/controller_health_check/controller_health_check.go

```diff
@@ -23,7 +24,22 @@ func CmdMain() {
 	if err := ovs.CheckAlive(); err != nil {
 		os.Exit(1)
 	}
-	conn, err := net.DialTimeout("tcp", "127.0.0.1:10660", 3*time.Second)
+
+	addr := "127.0.0.1:10660"
```

Comment:
health_check should only bind to localhost

Reply:
if kube-ovn-controller bind local IP, eg 172.18.0.2:10660,   the controller health check service will get the fault "connection refused " when accessing the 127.0.0.1 10660

Reply:
ok